cc_library/src/scripts/controlboard_config.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "controlBoard1",
+  "description": "Control board with three switches, three slides with lights and one main switch.",
+  "host": "192.168.178.82",
+  "input_components": [
```

Comment:
in GH-22 this was changed to be ```"input": { "<component_id>": "<"type">}``` instead of keeping components and input sepearate. Same for output

Reply:
the manual for devcie should also be changed